src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java14/InputJava14TextBlocks.java

```diff
@@ -84,4 +156,17 @@ public String getEscapedSpaces() {
             line 2·······\s
             """;
     }
+
+    void lineTerminators() {
+        String s = """
+                \u000D""";
+        String s1 = """
+                \n""";
+        String s2 = """
+                \n
+                """;
+        String s3 = """
+                \
+                """;
```

Comment:
looks good, I really hope visualization of "\n" if issue (by design) of our AST printer.
```
`--TEXT_BLOCK_LITERAL_BEGIN -> """ [161:19]
    |--TEXT_BLOCK_CONTENT -> \n                \u000D [161:22]
    `--TEXT_BLOCK_LITERAL_END -> """ [162:22]


`--TEXT_BLOCK_LITERAL_BEGIN -> """ [163:20]
    |--TEXT_BLOCK_CONTENT -> \n                \n [163:23]
    `--TEXT_BLOCK_LITERAL_END -> """ [164:18]

`--TEXT_BLOCK_LITERAL_BEGIN -> """ [165:20]
    |--TEXT_BLOCK_CONTENT -> \n                \n\n                 [165:23]
    `--TEXT_BLOCK_LITERAL_END -> """ [167:16]

`--TEXT_BLOCK_LITERAL_BEGIN -> """ [168:20]
    |--TEXT_BLOCK_CONTENT -> \n                \\n                 [168:23]
    `--TEXT_BLOCK_LITERAL_END -> """ [170:16]

```